packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -21,32 +21,33 @@ import {
 } from "@/components";
 import { TokenFromTokenlist } from "@/models/token";
 import { useAccount, useBalance, useNetwork } from "wagmi";
-import { formatUnits } from "viem";
+import { formatUnits, parseUnits } from "viem";
 import { ModalId, useModalContext } from "@/contexts";
+import { FREQUENCY_OPTIONS } from "@/models/stack";
+import { BigNumber } from "ethers";
```

Comment:
can't we use native `BigInt` ?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

viem docs: https://viem.sh/docs/ethers-migration.html#bignumber-utilities

Reply:
It will be handier if we leave `@ethers` lib

Reply:
We actually can, thanks!